controllers/clusterresources/redisuser_controller.go

```diff
@@ -205,11 +222,11 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 
 				return models.ReconcileRequeue, nil
 			}
+			continue
 		}
 
-		userID := fmt.Sprintf(instaclustr.RedisUserIDFmt, clusterID, username)
 		err = r.API.UpdateRedisUser(user.ToInstAPIUpdate(password, userID))
-		if err != nil {
+		if err != nil && !errors.Is(instaclustr.NotFound, err) {
```

Comment:
please switch the parameters in `!errors.Is`. Fix in similar cases

Reply:
fixed